crates/biome_js_analyze/tests/specs/nursery/useExportsLast/invalid.js

```diff
@@ -0,0 +1,3 @@
+export const a = 1;
```

Comment:
Could you check the original test file to add some more cases?
https://github.com/import-js/eslint-plugin-import/blob/main/tests/src/rules/exports-last.js